src/components/categories/Categories.css

```diff
@@ -0,0 +1,21 @@
+/* --------------------------- || CATEGORY SECTION ----------------- */
+.category-section {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  gap: 1rem;
+}
+.card-vertical {
+  min-width: 14rem;
+}
+.card-overlay-details {
+  background-color: var(--card-overlay-bg-color);
+}
+
+@media screen and (max-width: 1000px) {
```

Comment:
You can use em instead of px

Reply:
I agree but eventually it converts to `px` so for now i will go with this approach.